src/app/icd/BUILD.gn

```diff
@@ -25,6 +25,18 @@ source_set("manager-srcs") {
     "ICDManager.h",
   ]
 
+  public_deps = [
```

Comment:
Again, are these supposed to be `public_deps`?  Looks to me like `credentials:credentials` should be a public dep (since our public header includes that header), but `:monitoring-table` should not be (since we just include it in implementation).